src/coreclr/jit/importer.cpp

```diff
@@ -14159,11 +14317,13 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                         op1->gtFlags |= (GTF_OVERFLOW | GTF_EXCEPT);
                     }
 
+#ifndef TARGET_LOONGARCH64
```

Comment:
Please remove this `ifndef`. There should be no architecture-specific code in the importer.

Reply:
had removed.
Thanks